test/ccapi/unittest_ccapi.cpp

```diff
@@ -148,9 +148,9 @@ TEST(ccapi_dataset, construct_01_n) {
 /**
  * @brief Neural Network Dataset Contruct Test
  */
-TEST(ccapi_dataset, construct_02_p) {
-  EXPECT_NO_THROW(ml::train::createDataset(ml::train::DatasetType::GENERATOR));
-  EXPECT_NO_THROW(ml::train::createDataset(ml::train::DatasetType::FILE));
+TEST(ccapi_dataset, construct_02_n) {
+  EXPECT_THROW(ml::train::createDataset(ml::train::DatasetType::GENERATOR),
```

Comment:
Add the corresponding test for DatasetType::FILE?

Reply:
It is valid to create without argument for FILE so there is no such like this case.

Reply:
so the check of having file is added in the finalize of the Dataset?

Reply:
Correct 👍